InitializeRepo/initialize.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
```

Comment:
```suggestion

set -euo pipefail

```

These flags will, in order:
- Stop the script if any of the steps return an error (which prevents it erroring out later on resulting in a red herring)
- Stop the script if any referenced variables are empty
- Stop the script if an error occurs in a piped operation (which `-e` doesn't cover)

I'd encourage this line to always be your first in a bash script.

Reply:
Done